Assets/code_src/TimedAction.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections;
```

Comment:
A TimedAction is a single action that get's triggered after a certain amount of time. After it get's triggered, it's removed from the list of timer actions and never gets triggered again until you register it again.